tests/e2e/iot/device/main.c

```diff
@@ -0,0 +1,291 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#include <stdarg.h>

```

Comment:
Not sure why this, the mbedtls port, and tsconfig are showing up as new files instead of renames - might be worth figuring out to maintain the history.

Reply:
yea i tried forcing it but seems to keep them as new files and the adu ones as moved